src/encoded/static/components/antibody.js

```diff
@@ -12,14 +12,17 @@ import { DbxrefList } from './dbxref';
 import { DocumentsPanel, Document, DocumentPreview, CharacterizationDocuments } from './doc';
 import { RelatedItems } from './item';
 import { AlternateAccession } from './objectutils';
-import { StatusLabel } from './statuslabel';
+import { StatusLabel, getObjectStatuses, sessionToAccessLevel } from './statuslabel';
 
 
 const LotComponent = (props, reactContext) => {
     const context = props.context;
 
-    // Sort characterization arrays
-    const characterizations = _(context.characterizations).sortBy(characterization => (
+    // Sort characterization arrays, filtering for the current logged-in and administrative status.
+    const accessLevel = sessionToAccessLevel(reactContext.session, reactContext.session_properties);
+    const viewableStatuses = getObjectStatuses('AntibodyCharacterization', accessLevel);
+    let characterizations = context.characterizations.filter(characterization => viewableStatuses.indexOf(characterization.status) !== -1);
```

Comment:
Since we're using `babel-polyfill`, I think you should be able to reliably use `Array.includes`. However it's probably worth confirming that it works in IE11.